code/modules/mining/materials.dm

```diff
@@ -149,6 +166,18 @@ var/global/list/initial_materials	//Stores all the matids = 0 in helping New
 	cc_per_sheet=CC_PER_SHEET_GLASS
 	oretype=/obj/item/weapon/ore/glass
 	sheettype=/obj/item/stack/sheet/glass/glass
+	color = "#6E8DA2" //rgb: 110, 141, 162
+	alpha = 122
+	brunt_damage_mod = 0.7
+	sharpness_mod = 1.4
+
+/datum/material/glass/on_use(obj/source)
+	..()
```

Comment:
shouldn't the parent return something and shouldn't you check that something? as things are right now, calling the parent is useless